core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -21,6 +21,7 @@
 import java.util.Optional;
```

Comment:
In the commit message: "Include output format in plan json representation", did you mean "output columns", instead?

Reply:
Fixed the commit message